bdk-ffi/src/bdk.udl

```diff
@@ -116,260 +105,20 @@ enum WordCount {
-};
-
-dictionary ScriptAmount {
-  Script script;
-  u64 amount;
-};
-
-interface Wallet {
+interface Address {
   [Throws=BdkError]
-  constructor(Descriptor descriptor, Descriptor? change_descriptor, Network network, DatabaseConfig database_config);
+  constructor(string address, Network network);
 
   Network network();
 
-  [Throws=BdkError]
-  AddressInfo get_address(AddressIndex address_index);
-
-  [Throws=BdkError]
-  AddressInfo get_internal_address(AddressIndex address_index);
-
-  [Throws=BdkError]
-  boolean is_mine(Script script);
-
-  [Throws=BdkError]
-  sequence<LocalUtxo> list_unspent();
-
-  [Throws=BdkError]
-  sequence<TransactionDetails> list_transactions(boolean include_raw);
-
-  [Throws=BdkError]
-  Balance get_balance();
-
-  [Throws=BdkError]
-  boolean sign([ByRef] PartiallySignedTransaction psbt, SignOptions? sign_options);
-
-  [Throws=BdkError]
-  void sync([ByRef] Blockchain blockchain, Progress? progress);
-};
-
-interface FeeRate {
-  [Name=from_sat_per_vb]
-  constructor(float sat_per_vb);
-
-  float as_sat_per_vb();
-};
-
-dictionary SignOptions {
-  boolean trust_witness_utxo;
-  u32? assume_height;
-  boolean allow_all_sighashes;
-  boolean remove_partial_sigs;
-  boolean try_finalize;
-  boolean sign_with_tap_internal_key;
-  boolean allow_grinding;
-};
-
-interface Transaction {
-  [Throws=BdkError]
-  constructor(sequence<u8> transaction_bytes);
-
-  string txid();
-
-  u64 weight();
-
-  u64 size();
-
-  u64 vsize();
-
-  sequence<u8> serialize();
-
-  boolean is_coin_base();
-
-  boolean is_explicitly_rbf();
-
-  boolean is_lock_time_enabled();
-
-  i32 version();
-
-  u32 lock_time();
-
-  sequence<TxIn> input();
-
-  sequence<TxOut> output();
-};
-
-interface PartiallySignedTransaction {
-  [Throws=BdkError]
-  constructor(string psbt_base64);
-
-  string serialize();
-
-  string txid();
-
-  Transaction extract_tx();
-
-  [Throws=BdkError]
-  PartiallySignedTransaction combine(PartiallySignedTransaction other);
-
-  u64? fee_amount();
-
-  FeeRate? fee_rate();
-
-  string json_serialize();
-};
-
-dictionary TxBuilderResult {
-  PartiallySignedTransaction psbt;
-  TransactionDetails transaction_details;
-};
-
-interface TxBuilder {
-  constructor();
-
-  TxBuilder add_recipient(Script script, u64 amount);
-
-  TxBuilder add_unspendable(OutPoint unspendable);
-
-  TxBuilder add_utxo(OutPoint outpoint);
-
-  TxBuilder add_utxos(sequence<OutPoint> outpoints);
-
-  TxBuilder do_not_spend_change();
-
-  TxBuilder manually_selected_only();
-
-  TxBuilder only_spend_change();
-
-  TxBuilder unspendable(sequence<OutPoint> unspendable);
-
-  TxBuilder fee_rate(float sat_per_vbyte);
-
-  TxBuilder fee_absolute(u64 fee_amount);
-
-  TxBuilder drain_wallet();
-
-  TxBuilder drain_to(Script script);
-
-  TxBuilder enable_rbf();
-
-  TxBuilder enable_rbf_with_sequence(u32 nsequence);
-
-  TxBuilder add_data(sequence<u8> data);
-
-  TxBuilder set_recipients(sequence<ScriptAmount> recipients);
+  string to_qr_uri();
```

Comment:
I'm cool leaving this in but I think it is unused at the moment I think?

Reply:
Oh this might be a weird issue of how it looks in the GitHub UI I think (which makes it appear like the `to_qr_uri()` is part of the TxBuilder), but it's actually part of the `Address` type. So yes it's used.